packages/local_auth/ios/Classes/LocalAuthPlugin.m

```diff
@@ -5,13 +5,17 @@
 
 #import "LocalAuthPlugin.h"
 
-@implementation LocalAuthPlugin
+@implementation LocalAuthPlugin {
+  NSDictionary *lastCallArgs;
+  FlutterResult lastResult;
```

Comment:
I am not entirely sure about this. Saving these ephemeral states across app lifecycle seems wrong. If you have a better idea on how to do this, I am all ears :-).

Reply:
I agree it's ugly, but I don't have any recommendations 🤔